powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/behavior/tasks/OperationServiceBehavior.java

```diff
@@ -607,8 +612,10 @@ public OperationListResponse findAllOperationsForUser(final OperationListRequest
             throw localizationProvider.buildExceptionForCode(ServiceError.INVALID_APPLICATION);
         }
 
+        final Optional<List<String>> activationFlags = fetchActivationFlags(request.activationId);
```

Comment:
❗ Optional collection is usually antipattern. https://stackoverflow.com/a/68845910/204950